### DISCUSSION TITLE
Application not getting input

### DISCUSSION THREAD

**Discussion by CrusaderOne** [AntiMicroX/antimicrox#921] [General] [answered]
Hello oh wonderful people of the inter web.

I have Ubuntu 22.04 with Jack, Sooperlooper and AntimicroX 
I have mapped buttons to keys, but Sooperlooper does not seem to be getting the keystrokes? 
What might I be doing incorrectly?

Regards,
Neil

**Comment by pktiuk** [comment]
Do you have deb, or flatpak package?

**Reply by CrusaderOne** [reply]
I installed it via the software application on Ubuntu ?
So I assume deb ...

**Reply by pktiuk** [reply]
Don't assume, just check it (Help->About->Info)

**Reply by CrusaderOne** [reply]
Program Version 3.1.4
compiled from packaging: 3.1.4-1 (Ubuntu hirsute/amd64)
Built Against SDL 2.0.14
Running With SDL 2.0.20
Using Qt 5.15.3
Using Event Handler: XTest

**Reply by pktiuk** [reply, marked as answer]
This is an ancient version from 2 maybe 3 years ago.

**Reply by CrusaderOne** [reply]
ok, thanks, I'll try again.

**Reply by CrusaderOne** [reply]
thank you very much for your help.
I installed the 3.3.4 .deb package and it now works.

Regards,
Neil
